src/components/TagDropdown.tsx

```diff
@@ -88,7 +87,7 @@ export default function TagDropdown(props: TagDropdownProps) {
     
     return (
         <div className="flex flex-col p-4 gap-[10px]">
-            <input onClick={handleClick} onChange={handleSearch} value={searchInput} placeholder="tag name" className="flex flex-col justify-center pl-2 border-2 rounded-lg h-[46px]" />
+            <input onClick={handleClick} onInput={handleSearch} value={searchInput} placeholder="tag name" className="flex flex-col justify-center pl-2 border-2 rounded-lg h-[46px] text-black" />
```

Comment:
I'm not an expert, but I _think_ we want to run the search `onInput` rather than `onChange`